discovery-infra/test_infra/helper_classes/config/base_entity_config.py

```diff
@@ -24,3 +25,5 @@ class BaseEntityConfig(_BaseConfig, ABC):
     is_ipv6: bool = None
     base_dns_domain: str = None
     entity_name: BaseName = None
+    proxy: models.Proxy = None
```

Comment:
Why do we need it here? Why not pass it when it's needed

Reply:
It's used both in Cluster and Infraenv configs. but i'll add it to each explicitly

Reply:
@eliorerz moved back into BaseEntityConfig